src/otx/recipes/stages/detection/incremental.py

```diff
@@ -21,6 +21,12 @@
         interval=1,
         priority=75,
     ),
+    dict(
+        type="EMAHook",
+        priority="ABOVE_NORMAL",
+        resume_from=None,
```

Comment:
If model resumes training, then will this value be changed well?

Reply:
It will be solved by [c4a9ccc](https://github.com/openvinotoolkit/training_extensions/pull/2430/commits/c4a9ccc105d4bc9ffa085ce35d014f77d946c02a)
Could you check it? A unit test for this is also added